MAUI/Backdrop/reveal-height-customization.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Reveal height customization of Syncfusion Backdrop Page | Syncfusion
+description: Learn here all about how to configure and customize the reveal height in the Syncfusion backdrop page control.
+platform: MAUI
+control: SfBackdropPage
+documentation: ug
+---
+
+# Reveal Height Customization .NET MAUI Backdrop Page (SfBackdropPage)
+
+When revealing the `back layer`, the front layer will be moved downwards. By setting the `BackLayerRevealOption` property of backdrop, you can customize how far the front layer can be moved from the header when revealing the back layer.
+
+The following options are provided to move the front layer:
+
+`Fill` – Moves the front layer downwards to the entire height of the page excluding the `RevealedHeight`.
```

Comment:
`Fill` – Moves the front layer downwards to the entire height of the page, excluding the `RevealedHeight.`

Reply:
made changes